tests/integrate_test/test_samples/test_builtin/latest_version/governance/governance.py

```diff
@@ -754,6 +758,14 @@ def isInImportWhiteList(self, importStmt: str) -> bool:
             Logger.debug(f'({importStmt}) is in import white list')
         return True
 
+    @external(readonly=True)
+    def getImportWhiteList(self) -> list:
```

Comment:
I think it's better to return a list of `import_stmt` because the inputs of `addImportWhiteList` and `removeImportWhiteList`, `isInImportWhiteList` are `import_stmt`. This is minor opinion.